lesson4/Lesson4HW/app/src/main/java/com/margaret/lesson4hw/TasksAdapter.java

```diff
@@ -25,9 +25,7 @@
     @BindView(R.id.tvText) TextView tvText;
     private ArrayList<Task> tasks;
 
-    DictionaryOpenHelper mDbHelper = new DictionaryOpenHelper(getContext());
-    // Gets the data repository in write mode
-    final SQLiteDatabase db = mDbHelper.getWritableDatabase();
+    private DictionaryOpenHelper mDbHelper = new DictionaryOpenHelper(getContext());
```

Comment:
Make this private here and you don't need to declare `mDbHelper` down below on line 52!